packages/contracts-bedrock/test/L1/OptimismPortal.t.sol

```diff
@@ -24,13 +24,17 @@ import { OptimismPortal } from "src/L1/OptimismPortal.sol";
 contract OptimismPortal_Test is CommonTest {
     address depositor;
 
-    function setUp() public override {
+    /// @notice Marked virtual to be overridden in
```

Comment:
Nit, looks like we should use `@dev` for all the natspec tags added in the files in this PR, instead of `@notice`. To be fair I think we're inconsistent here across test files, but may as well use `dev` for consistency within these files

Reply:
My general reasoning was adding `@dev` to describe functions and `@notice` to note things, but I might have applied this a bit loosely. I don't have any preference and would generally vote for these files to be consistent with the usage of these tags in the rest of the monorepo (or at least, `contracts-bedrock`). Let me know what works best and I'll be sure to change it :+1:

Reply:
We can leave as-is, don't want to spend too much time bikeshedding nits. and I can always tweak in a follow up PR :)